dimod/include/dimod/quadratic_model.h

```diff
@@ -480,6 +260,17 @@ class QuadraticModelBase {
 
     QuadraticModelBase() : offset_(0) {}
 
+    /// Add quadratic bias for the given variables.
+    void add_quadratic(index_type u, index_type v, bias_type bias) {
+        assert(0 <= u && static_cast<size_t>(u) < this->num_variables());
```

Comment:
I generally write it as (assert u >= 0 && ..... ) 
The second part of your condition is like that too.

Reply:
Haha, I am used to python style `assert 0 <= u < self.num_variables())`